src/components/UserJourneyControls.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import useBoop from 'use-boop';
+import { animated } from 'react-spring';
+import { css } from '@emotion/react';
+import { Surface, Icon } from '@newrelic/gatsby-theme-newrelic';
+import SurfaceLink from './SurfaceLink';
+
+const UserJourneyControls = ({ previousStep, nextStep, className }) => {
+  const [rightButton, triggerRight] = useBoop({
+    x: 20,
+    springConfig: {
+      mass: 3,
+      tension: 160,
+      friction: 15,
+    },
+  });
+  const [leftButton, triggerLeft] = useBoop({
+    x: -20,
+    springConfig: {
+      mass: 3,
+      tension: 160,
+      friction: 15,
+    },
+  });
+  return (
+    <JourneyContainer>
+      <JourneyLink
+        base={Surface.BASE.SECONDARY}
+        to={previousStep.path}
+        interactive
+        instrumentation={{
+          category: 'UserJourney',
+          eventName: 'previousStepClick',
+          path: previousStep.path,
+        }}
+        className={className}
+        onMouseEnter={triggerLeft}
+      >
+        <InnerContainer>
+          <animated.div style={leftButton}>
+            <Icon
+              name="fe-arrow-left"
+              size="3rem"
+              css={css`
+                color: var(--system-text-primary);
+                margin-right: 2rem;
+                .doc-tiles-labs & {
+                  display: none;
+                }
```

Comment:
is this needed since this is separate from the DocTiles and doesn't have variants? same on line 94